lldb/test/API/api/multiple-debuggers/multi-process-driver.cpp

```diff
@@ -241,26 +257,13 @@ int main (int argc, char **argv)
     }
 
 
-    int max_time_to_wait = 20;  // 20 iterations, or 60 seconds
-    int iter = 0;
-    while (1)
-    {
+    int max_time_to_wait = 40;  // 40 iterations, or 120 seconds
```

Comment:
We usually multiply all timeouts by a factor 10 when running under ASAN. A reasonably proxy for that would be if getenv("ASAN_OPTIONS") returns a nonempty string, because lit sets that environment variable.